### ISSUE TITLE
enabled SpaceBetweenParens doesn't allow parameter-less functions

### ISSUE THREAD

**Issue by montchr** [brigade/scss-lint#552]
Configuration:

```yml
  SpaceBetweenParens:
    enabled: true
    spaces: 1
```

If I try to run a function like `function()` without a parameter, SpaceBetweenParens throws an error. If I add spaces, I still receive a combination of errors and will always fail because it never detects the proper number of spaces between each parenthesis. I've tried 2 and 3 spaces between parens, neither of which work.

Potentially related to #475.

**Comment by sds** [comment]
Fixed by #558.

**Comment by montchr** [comment]
This is still an issue for me even after updating to 0.41.0…

The first line and the line beginning with "offset" throw an error:

```scss
$c-title-bar-settings: (
  title-font-weight     : map-get-strict( $o-blox-settings, 'title-font-weight' ),
  title-font-weight-alt : map-get-strict( $o-blox-settings, 'title-font-weight-alt' ),
  font-sizes            : $c-title-bar-font-sizes,
  pd-v                  : $half-spacing-unit,
  pd-h                  : ( map-get-strict( $c-title-bar-defaults, 'item-spacing' ) * 2 ),
  item-spacing          : map-get-strict( $c-title-bar-defaults, 'item-spacing' ),
  offset                : o-blox-offset-get(),
  promo-bg              : $hello-yellow
) !default;
```

But this does not throw an error:

```scss
$c-title-bar-settings: map-merge( $c-title-bar-settings, (
  m-t: ( o-blox-height-get() + $base-spacing-unit )
) );
```
